litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -530,6 +530,14 @@ spec:
       labels:
         component: litmusportal-server
     spec:
+      initContainers:
+        - name: wait-for-mongodb
+          image: curlimages/curl:7.78.0
```

Comment:
We need to probably put this image in litmuschaos docker repo, but that can be done before the release

Reply:
Yes, will have to do that before release.